rest/model/host.go

```diff
@@ -43,6 +43,18 @@ type APIHost struct {
 	CreationTime          *time.Time  `json:"creation_time"`
 	Expiration            *time.Time  `json:"expiration_time"`
 	AttachedVolumeIDs     []string    `json:"attached_volume_ids"`
+	// Contains options for spawn hosts.
+	ProvisionOptions APIProvisionOptions `json:"provision_options"`
+}
+
+// APIProvisionOptions contains options for spawn hosts.
+type APIProvisionOptions struct {
+	// ID of the task that the host was spawned from.
+	TaskID *string `json:"task_id"`
```

Comment:
How is this TaskID different than the task id in https://github.com/evergreen-ci/evergreen/pull/8095/files#diff-ecc3ce0d8374bf1432eea71ccd38db5be0de0319ace2df7c07337ca0091a8a6dR33 (line 33 of this file, field called RunningTask)

Reply:
That's the current running task ID of a task host, which changes over time. The spawn host task ID is for the task that the spawn host came from (when you click "Spawn Host" on the task page) and is used to determine what artifacts to load on the host (if the user asks for it).